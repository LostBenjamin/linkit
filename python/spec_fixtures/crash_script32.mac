
[macro 4]
  (((waste 4)))

[macro2 5]
  red

[[waste INSTANCE
  12345,12345,12345
  sto
]]

