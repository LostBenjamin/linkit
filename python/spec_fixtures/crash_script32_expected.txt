No block of macros for fixed macro during final number assignment

